src/HealthChecks.InfluxDB/DependencyInjection/InfluxDBHealthCheckBuilderExtensions.cs

```diff
@@ -0,0 +1,198 @@
+using HealthChecks.InfluxDB;
+using InfluxDB.Client;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.DependencyInjection

```

Comment:
Please use file-scoped namespaces for all new code.

Reply:
OK